kubetest2-gke/deployer/up.go

```diff
@@ -65,12 +65,14 @@ func (d *deployer) Up() error {
 	defer cancel()
 	eg, ctx := errgroup.WithContext(ctx)
 	loc := location(d.region, d.zone)
+	clusterIndex := 0
 	for i := range d.projects {
 		project := d.projects[i]
 		subNetworkArgs := subNetworkArgs(d.projects, d.region, d.network, i)
 		for j := range d.projectClustersLayout[project] {
 			cluster := d.projectClustersLayout[project][j]
-			privateClusterArgs := privateClusterArgs(d.network, cluster, d.privateClusterAccessLevel, d.privateClusterMasterIPRange)
+			privateClusterArgs := privateClusterArgs(d.network, cluster, d.privateClusterAccessLevel, d.privateClusterMasterIPRanges[clusterIndex])
```

Comment:
do we want to retain the 1:1 mapping of the clusters input and private master range input since they are tightly coupled?
currently it would assign the private ip range based the iteration order of the map.

Reply:
I have updated the logic to retain the 1:1 mapping. Thanks!